charts/dshackle/templates/_helpers.tpl

```diff
@@ -60,3 +60,10 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{/*
+HTTP port
```

Comment:
HTTPS?

Reply:
We could also change to HTTPS, but throughout this chart `http` is used to label that specific traffic

Reply:
`http` here represents internal port which is not HTTPS, internally they communicate over HTTP on port 8545